shell/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# mcr.microsoft.com/azurelinux/base/core:3.0.20241005
-FROM mcr.microsoft.com/azurelinux/base/core@sha256:7ec490b605aac8a44aed0b0695b0ee6ae976ec898afd9ac8d5613d7f3ce2b07b
+# mcr.microsoft.com/azurelinux/base/core:3.0.20241101
```

Comment:
What purpose is this line serving?

Reply:
since we are pinning the images by sha256 tags, it might be helpful to indicate which of the actual image the tag belongs to

Reply:
Thanks, makes sense